openmrs-19/src/main/java/org/motechproject/openmrs19/tasks/OpenMRSActionProxyService.java

```diff
@@ -100,4 +100,14 @@ void updatePerson(String personUuid, String givenName, String middleName, String
                       DateTime startDate, DateTime endDate, DateTime birthdate, Boolean birthdateEstimated,
                       String gender, Boolean dead, String causeOfDeathUUID);
 
+    /**
+     * Creates a program enrollment with the given params.
+     * The required fields are: {@code patientUuid}, {@code programUuid}, {@code dateEnrolled}.
+     *
+     * @param patientUuid the patient uuid
+     * @param programUuid the program uuid
+     * @param dateEnrolled the program enrollment date
+     * @param dateCompleted the program completed date
+     */
+    void createProgramEnrollment(String patientUuid, String programUuid, DateTime dateEnrolled, DateTime dateCompleted);
```

Comment:
I see that ProgramEnrollment has also states and location, what about it ? Can you write why don't you use this fields in task action.